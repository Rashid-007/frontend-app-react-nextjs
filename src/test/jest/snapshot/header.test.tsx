	
import React from "react";
import renderer from "react-test-renderer";

import Index from "../../../main/pages/index";

describe("Login", () => {
  it("renders the h1 title", () => {
    const index = renderer.create(<Index />).toJSON();
    expect(index).toMatchSnapshot();
  });
})