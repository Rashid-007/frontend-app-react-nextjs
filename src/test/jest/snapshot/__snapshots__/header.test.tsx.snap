// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders the h1 title 1`] = `
Array [
  .emotion-3 {
  background: #5a95bb;
}

.emotion-2 {
  margin: 0 auto;
  max-width: 1200px;
  height: 100px;
  padding: 30px;
}

.emotion-1 {
  margin: 0;
}

.emotion-0 {
  color: white;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
}

<header
    className="emotion-3"
  >
    <div
      className="emotion-2"
    >
      <h1
        className="emotion-1"
      >
        <a
          className="emotion-0"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          SHOP
        </a>
      </h1>
    </div>
    <nav>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Home
      </a>
       
       
    </nav>
  </header>,
  .emotion-0 {
  margin: 0 auto;
  max-width: 1200px;
  padding: 0px 1.5em 3.0em;
}

<div
    className="emotion-0"
  >
    <main>
      <h1>
        index.hi
         ðŸ‘‹
      </h1>
      <h2>
        Hello world! APP_ENV: 
         and cms-port: 
      </h2>
      <p>
        index.browser-dimensions.headline
      </p>
      <ul>
        <li>
          index.browser-dimensions.height
           
          768
        </li>
        <li>
          index.browser-dimensions.width
           
          1024
        </li>
      </ul>
    </main>
    <footer>
      Â© 
      2019
    </footer>
  </div>,
]
`;
